tollway/constants.py

```diff
@@ -2,14 +2,14 @@
 
 from us.states import STATES_AND_TERRITORIES
 
-from tollway.utils import get_envs
+from tollway.utils import get_config_with_default
 
 STATE_NAMES = [state.name for state in STATES_AND_TERRITORIES]
 URL_PATH = "wiki/List_of_toll_roads_in_the_United_States"
 TOLLWAYS_URL = f"https://en.wikipedia.org/{URL_PATH}"
 TIMESTAMP_FORMAT = "%Y-%m-%d %H:%M:%S.%f %z"
-DUPLICATE_RATE = get_envs("duplicate")
-ALL_EVENTS_COUNT = get_envs("all")
+DUPLICATE_RATE = get_config_with_default(key="DUPLICATE_RATE", default_value="50")
```

Comment:
should these be ints?

Reply:
in the `events` module DUPLICATE_RATE and LATE_EVENT_RATE[...] are being compared to len which is why I'm casting to `int` here. Suppose I could leave this as string and then cast as `int` in the `events` module. I would also need to cast ALL_EVENTS_COUNT in `__main__` when comparison occurs. 

<img width="722" alt="Screenshot 2024-02-14 at 4 38 45 PM" src="https://github.com/BenGriffith/tollway-traffic/assets/6089155/7a305cbd-5b32-45eb-8d67-c444cf820709">

Reply:
yes, that's it (please see comments below related to this one). When there is no .env file, I started by using ints like `50` for default_value, however, I then was receiving an error for the value.strip() as seen in screenshot here so I changed from `int` to `str`.

<img width="1975" alt="Screenshot 2024-02-14 at 5 54 30 PM" src="https://github.com/BenGriffith/tollway-traffic/assets/6089155/ebe8599f-bbe2-4c7d-b55f-90519e92e86a">

Reply:
this snippet could work as an alternative
<img width="662" alt="Screenshot 2024-02-14 at 6 07 55 PM" src="https://github.com/BenGriffith/tollway-traffic/assets/6089155/48269a2a-4601-4d89-b10d-2edfcedd8d7c">

Reply:
scratch that suggestion. Think I'm going to go with this one - so pass an int to the default_value of the wrapper and then in the wrapper remove .strip() and do the int(value) cast. Shown in screenshot.

<img width="1583" alt="Screenshot 2024-02-14 at 7 32 28 PM" src="https://github.com/BenGriffith/tollway-traffic/assets/6089155/b2e22308-f358-4374-af0e-46560bd62094">

Reply:
Maybe we can revisit this when we meet ...